pkg/storage/disk/all_test.go

```diff
@@ -0,0 +1,41 @@
+package disk
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	baseURL = "testbaseurl"
+	module  = "testmodule"
+	version = "v1.0.0"
+)
+
+var (
+	mod = []byte("123")
```

Comment:
maybe a note for future us that a wider variety of file sizes would be useful to catch possible edge cases - especially related to http and buffers, etc.